isis/tests/DistanceTests.cpp

```diff
@@ -0,0 +1,295 @@
+#include "Distance.h"
+#include "Displacement.h"
+#include "IException.h" 
+#include "SpecialPixel.h"
+
+#include <gtest/gtest.h>
+
+using namespace Isis;
+
+TEST(DistanceTests, DefaultConstructor) {
+	Distance dist;
+	EXPECT_EQ(dist.meters(), Null);
+	EXPECT_EQ(dist.kilometers(), Null);
+	EXPECT_EQ(dist.pixels(), Null);
+	EXPECT_EQ(dist.solarRadii(), Null);
+}
+
+
+TEST(DistanceTests, MetersConstructor) {
```

Comment:
Using parameters would would make these nicer, but I don't think these could be shortened into one test since the order of returning the distance in a specific unit would be different for each test.